crates/wkg/src/warg.rs

```diff
@@ -0,0 +1,23 @@
+use clap::Args;
+
+#[derive(Args, Debug)]
+pub struct GetArgs {
+    // TODO: Args
```

Comment:
@calvinrp I could use your help implementing the warg part of this since you'll probably be quicker than me for defining the right set of flags and what not. If you can't get to it, I'll get to it next week